app/src/main/java/com/iron/espresso/model/response/PlaceResponse.kt

```diff
@@ -0,0 +1,57 @@
+package com.iron.espresso.model.response
+
+import com.google.gson.annotations.SerializedName
+
+data class PlaceResponse(
+    @SerializedName("meta")
+    val meta: PlaceMeta,
+    @SerializedName("documents")
+    val documents: List<Place>
+)
+
+data class PlaceMeta(
+    @SerializedName("total_count")
+    val totalCount: Int,
+    @SerializedName("pageable_count")
+    val pageableCount: Int,
+    @SerializedName("is_end")
+    val isEnd: Boolean,
+    @SerializedName("same_name")
+    val sameName: RegionInfo
+)
+
+data class RegionInfo(
+    @SerializedName("region")
+    val region: List<String>,
+    @SerializedName("keyword")
+    val keyword: String,
+    @SerializedName("selected_region")
+    val selected_region: String
+)
+
+data class Place(
+    @SerializedName("id")
+    val id: String,
+    @SerializedName("place_name")
+    val place_name: String,
```

Comment:
```suggestion
    val placeName: String,
```
snake_case로 되어있는것들 camelCase 로 변경하는게 좋겠어요

Reply:
수정했어요 앞으로 신경쓰도록 하겠습니다